src/app/page.tsx

```diff
@@ -1,113 +1,29 @@
-import Image from "next/image";
+"use client";
```

Comment:
It's not a best idea to mark "use client" the entire page. It's better to create a separate component file and extract client logic there

Reply:
It is kinda leftover, only to test component library, it will be removed before merge

Reply:
ok, resolve this conversation after deleting it, and everything will be ok